docs/src/components/Hero/SelectedLabel/index.tsx

```diff
@@ -1,38 +1,191 @@
 import clsx from 'clsx';
-import React from 'react';
+import React, { useEffect, useRef, useState } from 'react';
 import styles from './styles.module.css';
+import SelectionBox, { DraggableId } from './SelectionBox';
+
+const SelectedLabel: React.FC<{
+  children: React.ReactNode;
+  isInteractive: Boolean;
+}> = ({ children, isInteractive = false }) => {
+  const selectionRef = useRef(null);
+  const selectionContainerRef = useRef(null);
+  const textLabelRef = useRef(null);
+
+  const positionStyles = {
+    top: 0,
+    left: 0,
+    width: null,
+    height: null,
+  };
+  const setPositionStyles = (newPositionStyles: {
```

Comment:
It can be defined as a type {
    top: number;
    left: number;
    width: number;
    height: number;
  }